packages/client/src/components/Recipe/Recipe.tsx

```diff
@@ -0,0 +1,20 @@
+const Recipe = () => {
+  return (
+    <>
+      <div>picture</div>
+      <div>
+        <div>
+          recipe details
+          <section>
+            <div>Title</div>
+            <p>Description of Recipe</p>
+          </section>
+          <section>Instructions of Recipe</section>
+        </div>
+        <button>Delete this recipe</button>
+        <button>Favorite this recipe</button>
+      </div>
+    </>
+  );
+};
+export default Recipe;
```

Comment:
I know this isn't finalized yet, but a couple other things we will want to display include the time (in minutes) and the servings (number).